app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -86,9 +86,7 @@ protected void onCreate(final Bundle savedInstanceState) {
             LOGGER.error("Failed to retrieve contacts from database", e);
         }
 
-        sorter = new MainActivitySorter(this);
-        sorter.sortOnName();
```

Comment:
We do, but we also want to sort on name when the activity is started or resumed, that is why you can find `sorter.sortOnName()` in the [`onResume()` method](https://github.com/ericcornelissen/NervousFish/pull/223/files#diff-149538cc1f1f5e4370b6ae93d6489330R133)

You can see that this should work based on the [Android Activity Lifecycle diagram](https://developer.android.com/guide/components/images/activity_lifecycle.png) from the [Android documentation](https://developer.android.com/guide/components/activities/activity-lifecycle.html)